nova/network/neutronv2/api.py

```diff
@@ -1314,6 +1314,18 @@ def bind_ports_to_host(self, context, instance, host,
             if resp:
                 bindings_by_port_id[port_id] = resp.json()['binding']
             else:
+                # In case we actually managed to create the binding,
+                # but something went wrong in between, we can recover
+                if resp.status_code == 409:
+                    error_type = resp.json().get('NeutronError', {}).get('type')
+                    if error_type == 'PortBindingAlreadyExists':
+                        resp = client.get(
```

Comment:
With us overwriting `resp` here, we will get a confusing error-message, if the request fails for some reason. What's your opinion on doing an explicit `LOG.info`, that we will try to fetch the binding instead or something so we know were the `LOG.error` down below originated from?